### ISSUE TITLE
PetrovichHelper needs a refactoring

### ISSUE THREAD

**Issue by ghost** [petrovich/petrovich-java#1]
Helper does a lot of things. Need divide responsibility (algorithms, parsings, inner class).

**Comment by mulya** [comment]
Agree, but can't say when I will have enought free time for this refactoring.

**Comment by mulya** [comment]
By the way, feel free to make an pull request.
